helper/communicator/step_connect.go

```diff
@@ -53,6 +53,7 @@ func (s *StepConnect) Run(state multistep.StateBag) multistep.StepAction {
 			Config:      s.Config,
 			Host:        s.Host,
 			WinRMConfig: s.WinRMConfig,
+			WinRMPort:   s.SSHPort,
```

Comment:
I haven't looked at this very closely yet, but why does this use SSHPort? Is that coming from the config template? This might be correct code but the name looks out of place.